ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/JobImpl.java

```diff
@@ -142,4 +156,10 @@ protected TransferStrategyBuilder transferStrategyBuilder() {
     protected EventDispatcher eventDispatcher() {
         return transferStrategyBuilder.eventDispatcher();
     }
+
+    protected void cancel(final TransferStrategy transferStrategy) throws IOException {
+        if (transferStrategy != null) {
```

Comment:
We probably should log a warning when this is called but the transferstrategy is null.